numpy/lib/shape_base.py

```diff
@@ -255,6 +255,7 @@ def expand_dims(a, axis):
 
     See Also
     --------
+    squeeze : The inverse operation, removing singleton dimensions
```

Comment:
I personally like indexing a lot for this both of this too, unless it gets very clunky with counting `:`.